em_workflows/brt/flow.py

```diff
@@ -503,16 +503,16 @@ def gen_zarr(fp_in: FilePath):
 def gen_ng_metadata(fp_in: FilePath) -> Dict:
     file_path = fp_in
     asset_fp = Path(f"{file_path.assets_dir}/{file_path.base}.zarr")
-    zarr_fp = Path(f"{file_path.working_dir}/{file_path.base}.zarr")
-    hw_images = HedwigZarrImages(zarr_fp)
+    hw_images = HedwigZarrImages(asset_fp)
     _, hw_image = next(hw_images.series())
 
+    first_zarr_arr = Path(asset_fp.as_posix() + "/0/0")
```

Comment:
You should be able to use the ZarrImage.path to get the location of an image.

 The root of the pyramid should be "just under /0", not too 0.

Besides that, in general Path objects can be joined with the "/" operator. For example `path1 / path2` or `path / "dir"`.